packages/f-alert/src/components/Alert.vue

```diff
@@ -48,16 +50,19 @@ import {
     WarningIcon
 } from '@justeat/f-vue-icons';
 import { globalisationServices } from '@justeat/f-services';
+import FButton from '@justeat/f-button';
```

Comment:
Out of interest why not call this something like `DismissAlertButton` or similar. Would help with understanding at a glance what the button represents.

Reply:
Because if the use case arrose for two buttons to be required then this would invalidate that naming convention

Reply:
Ah that's my bad i just realised this is a component and not a consuming one. Sorry!